content/docs/2.10/scalers/prometheus.md

```diff
@@ -43,7 +43,7 @@ triggers:
 
 ### Authentication Parameters
 
-Prometheus Scaler supports four types of authentication - bearer authentication, basic authentication, TLS authentication and custom authentication.
+Prometheus Scaler supports five types of authentication - bearer authentication, basic authentication, TLS authentication, custom authentication and Azure Workload/Pod Identity for Azure managed service for Prometheus.
```

Comment:
Given the duplication, I think we can change this to:
```suggestion
Prometheus Scaler supports various types of authentication to help you integrate with Prometheus.
```

Reply:
done